src/main/java/com/ldtteam/structures/client/BlueprintBlockAccess.java

```diff
@@ -10,6 +10,9 @@
 import com.ldtteam.structures.blueprints.v1.Blueprint;
```

Comment:
My generic approach in this class was supply null whereever possible and then test and fix crashes according our needs.
Will clean it up myself in my next PR.